cmd/schema_gen.go

```diff
@@ -74,6 +88,62 @@ func generateSchemaForGraph(graph *fs.Graph) error {
 		pb.Increment()
 		return nil
 	}, config.NumberThreads(), pb)
+
+}
+
+type DocSuggestions struct {
+	mu          sync.Mutex
+	suggestions map[string][]string
+}
+
+func (d *DocSuggestions) AppendSuggestion(modelName string, modelSuggestions []string) {
+	d.mu.Lock()
+	d.suggestions[modelName] = modelSuggestions
+	d.mu.Unlock()
+}
+
+func (d *DocSuggestions) Init() {
+	d.mu.Lock()
+	d.suggestions = make(map[string][]string)
+	d.mu.Unlock()
+
+}
+
+func (d *DocSuggestions) Value() (suggestions map[string][]string) {
+	d.mu.Lock()
+	suggestions = d.suggestions
+	d.mu.Unlock()
+	return
+}
+
+func suggestDocsForGraph(graph *fs.Graph) error {
```

Comment:
We execute over a new graph, because we have to run after we run the original schema gen code. Is there is a more elegant way to avoid duplicating the graph.

We toyed with passing a slice of functions to the graph.Execute method - but felt this might be overkill.

Reply:
You could use the `graph.UnmarkFileAsRun`, which is what Watch mode uses to reset parts of the graph. So I'd suggest maybe creating a `graph.ResetRunStatus` method which just calls UnmarkFileAsrun for all files in the graph?

Reply:
Sounds good - I'll look into it!

Reply:
Added a new method and used it to unmark the graph run status ✅